latex/latex.tex

```diff
@@ -0,0 +1,287 @@
+\usection{Anhang B: Tipps für \LaTeX}\label{latex}
+
+Hier finden Sie gesammelte Hinweise, die sich speziell auf das \LaTeX-Template und dessen Verwendung beziehen. Der Aufbau des Templates wird dargelegt und wichtige Befehle werden anhand von Beispielen erklärt.
+
+\usubsection{Struktur des Templates}
+
+\begin{description}
+    \item[mi-document/]{Dieses Verzeichnis enthält die Style- und Class-Files, die das eigentliche Template darstellen. In diesen werden Pakete importiert, Formatierungen definiert und zusätzliche Funktionen bereitgestellt. Manche Entwicklungsumgebungen für \LaTeX, wie TeXmaker und TeXstudio, finden diese Dateien leider nur, wenn sie sich im gleichen Verzeichnis befinden wie \verb|document.tex|. In diesem Fall müssen alle \verb|.sty|- und \verb|.cls|-Dateien in dessen Verzeichnis verschoben werden.}
+    \item[document.tex]{Diese Datei enthält den Quellcode für dieses Dokument. Es wird empfohlen, Arbeiten auf diese Datei aufzubauen. In den ersten Zeilen werden Variablen definiert (z.B. Autor, Titel der Arbeit, etc.), welche vom Template beispielsweise für das Deckblatt und Metadaten verwendet werden. Danach werden Inhalts-, Abbildungs-, Tabellen- und Codeverzeichnis angezeigt. Der tatsächliche Inhalt der Arbeit kann direkt in diese Datei geschrieben werden, es wird jedoch empfohlen, Kapitel auf verschiedene Dateien aufzuteilen und über den \verb|\input|-Befehl einzubinden.}
```

Comment:
Vielleicht noch darauf hinweisen, dass der Name im Makefile geändert werden kann.

Reply:
Warum wird empfohlen, Kapitel in Dateien aufzuteilen? Kurze Begründung wäre gut.